### PR TITLE
feat: gcloud asdf tool

### PR DESCRIPTION
**Pull request by Pehesi97**
## What does this PR do?

This PR adds gcloud to the asdf tools.

## Checklist before merging

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have checked the [contributing document](../CONTRIBUTING.MD).
- [X] I have checked the existing [Pull Requests](https://github.com/nearform/initium-platform/pulls) to see whether someone else has raised a similar idea or question.
- [ ] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,6 +54,8 @@ plugin_uninstall: ## uninstall asdf plugins
 	@asdf plugin remove kubectl || true
 	@asdf plugin remove pre-commit || true
 	@asdf plugin remove tilt || true
+	@asdf plugin remove tilt || true
```
Comment (LucaLanziani) [inline Makefile:57]: I believe this is a duplicate

Comment (LucaLanziani) [inline Makefile:57]:
```suggestion
```

File: Makefile [hunk 2]
```diff
@@ -40,6 +40,7 @@ plugin_install: ## install asdf plugins
 	@asdf plugin add kubectl https://github.com/asdf-community/asdf-kubectl.git || true
 	@asdf plugin add pre-commit https://github.com/jonathanmorley/asdf-pre-commit.git || true
 	@asdf plugin add tilt https://github.com/virtualstaticvoid/asdf-tilt.git || true
+	@asdf plugin add gcloud https://github.com/jthegedus/asdf-gcloud || true
```
Comment (spchortis) [inline Makefile:43]: Maybe add .git suffix to make it consistent?

File: Makefile (hunk 1 above)
Comment (spchortis) [inline Makefile:57]: Please check if it is copied twice... Tilt seems to be already there..

File: Makefile (hunk 2 above)
Comment (LucaLanziani) [inline Makefile:43]:
```suggestion
	@asdf plugin add gcloud https://github.com/jthegedus/asdf-gcloud.git || true
```

**Review by LucaLanziani — APPROVED** [review]

**Comment by spchortis** [comment]
As agreed with Pedro we are going to close this one and keep [that](https://github.com/nearform/initium-platform/pull/197) one.

**Comment by spchortis** [comment]
As agreed with Pedro we are going to close this one and keep [that](https://github.com/nearform/initium-platform/pull/197) one.

**Pull request closed without merging**